apps/modernization-ui/src/permissions/useSystemManagementAccess.ts

```diff
@@ -0,0 +1,21 @@
+import { useMemo } from 'react';
+
+const SYSTEM_MANAGEMENT_PERMISSIONS = [
+    'EPILINKADMIN-SYSTEM',
+    'VIEWELRACTIVITY-OBSERVATIONLABREPORT',
+    'SRTADMIN-SYSTEM',
+    'VIEWPHCRACTIVITY-CASEREPORTING',
+    'IMPORTEXPORTADMIN-SYSTEM',
+    'REPORTADMIN-SYSTEM',
+    'ALERTADMIN-SYSTEM'
+];
+
+const useSystemManagementAccess = (permissions: string[] | undefined | null) => {
```

Comment:
I'm on the fence out passing the permissions vs using `useUser`

Reply:
your right we can directly use the useUser hook inside the custom hook